src/database/SsiCredentialIssuer.Migrations/Migrations/20240502065751_95-removeQualityCredential.cs

```diff
@@ -0,0 +1,68 @@
+using Microsoft.EntityFrameworkCore.Migrations;
```

Comment:
Could you please still add a file header?

Reply:
added